app/examples/qutebrowser

```diff
@@ -0,0 +1,111 @@
+" Options
+set adblocker=update
+set devtoolsposition=split
+set downloadmethod=confirm
+set firefoxmode=always
+set fontsize=16
+set norestoretabs
+set permissioncamera=ask
+set permissiondisplaycapture=ask
+set permissionmediadevices=allow
+set permissionmicrophone=ask
+set permissionpersistentstorage=allow
+set redirecttohttp
+set tabclosefocusright
+set tabreopenposition=next
+set windowtitle=full
+set mapsuggestposition=bottomleft
+
+" Mappings
+nmap <Up> <action.scrollUp>
+nmap <Down> <action.scrollDown>
+nmap <Left> <action.scrollLeft>
+nmap <Right> <action.scrollRight>
+nmap <PageUp> <action.scrollPageUp>
+nmap <PageDown> <action.scrollPageDown>
+nmap <Home> <action.scrollTop>
+nmap <End> <action.scrollBottom>
+
+nmap o <action.toExploreMode>
+nmap wi <:devtools>
+nmap <C-PageUp> <action.previousTab>
+nmap <C-PageDown> <action.nextTab>
+nmap <F5> <action.reload>
+nmap <C-F5> <action.reloadWithoutCache>
+nmap Sh <:history>
+nmap Ss <:help>
+nmap <C-A-p> <:print>
+nmap <C-q> <:quit>
+nmap <C-W> <:quit>
+nmap ZQ <:quit>
+nmap gd <:write>
+nmap <C-Tab> <action.nextTab>
+nmap <C-S-PageUp> <action.moveTabBackward>
+nmap <C-S-PageDown> <action.moveTabForward>
+nmap <C-S-Tab> <action.previousTab>
+nmap <C-I> <:devtools>
+nmap <C-T> <action.reopenTab>
+nmap <A-1> <:b 0>
+nmap g0 <:b 0>
+nmap g^ <:b 0>
+nmap g$ <:b 9999>
+nmap <A-2> <:b 1>
+nmap <A-3> <:b 2>
+nmap <A-4> <:b 3>
+nmap <A-5> <:b 4>
+nmap <A-6> <:b 5>
+nmap <A-7> <:b 6>
+nmap <A-8> <:b 7>
+nmap <A-9> <:b 9999>
+nmap <C-^> <action.toLastUsedTab>
+nmap <C-Tab> <action.toLastUsedTab>
+nmap ga <action.openNewTab>
+nmap gC <action.openNewTabWithCurrentUrl>
+nmap gJ <action.moveTabForward>
+nmap gK <action.moveTabBackward>
+nmap <C-w> <:close>
+nmap co <:lclose><:rclose>
+nmap = <action.zoomReset>
+nmap <C-p> <:pin>
+nmap <A-m> <:mute>
+nmap <C-s> <action.stopLoadingPage>
+nmap o <action.toExploreMode><BS>
+nmap go <action.toExploreMode><End>
+nmap gt <action.toCommandMode>buffer<Space>
+nmap O <action.openNewTab><action.toExploreMode>
+nmap yy <action.pageToClipboard>
+nmap pp <action.openFromClipboard>
+nmap Pp <action.openNewTab><action.openFromClipboard>
+nmap sk <action.toCommandMode>map<Space>
+nmap ss <action.toCommandMode>set<Space>
+nmap <C-S-Tab> <nop>
+
+imap <C-e> <action.editWithVim>
+
+cmap <Up> <action.commandHistoryPrevious>
+cmap <Down> <action.commandHistoryNext>
+cmap <C-a> <Home>
+cmap <C-e> <End>
+cmap <A-f> <C-Right>
+cmap <A-b> <C-Left>
+cmap <A-d> <C-S-Right><BS>
+cmap <A-BS> <C-S-Left><BS>
+cmap <C-?> <Del>
+cmap <C-h> <BS>
+cmap <C-b> <Left>
+cmap <C-f> <Right>
+cmap <C-k> <S-End><BS>
+cmap <C-u> <S-Home><BS>
+cmap <C-w> <C-S-Left><BS>
+
+pmap c <action.toNormalMode>
+pmap e <pointer.moveFastRight>
```

Comment:
I understand mapping this to e due to this being similar to Vim, so this is fine, but we should still provide the option to inspect at a position. Is there something similar in qutebrowser that we can map to a different key?

Reply:
There is only a command to open devtools ([devtools](https://qutebrowser.org/doc/help/commands.html#devtools)) and a command to focus devtools ([devtools-focus](https://qutebrowser.org/doc/help/commands.html#devtools-focus)). I don't know of any way to inspect at a position.

Reply:
In that case this is fine.